api-testing/multicloud-api/settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'multicloud-api'
```

Comment:
Is this a module? or project?

Reply:
Project.

Reply:
How will it handle for other projects in future example sodafoundation/api?

Can we refactor the name also multicloud-api -> multi-cloud?

Reply:
Do refactor, In intellije have option it will change everywhere for same name. [LINK](https://www.jetbrains.com/help/idea/refactoring-source-code.html)

Reply:
I didn't get the proper reasoning for the nomenclature.
Decide whether you need to keep  multicloud-api or multi-cloud ?

Reply:
Yes, multicloud-api